source/utilities.tex

```diff
@@ -16940,15 +16940,15 @@
 \tcode{template<bool B, class T = void>} \tcode{struct enable_if;}
  &
  If \tcode{B} is \tcode{true}, the member typedef \tcode{type}
- shall equal \tcode{T}; otherwise, there shall be no member
+ names \tcode{T}; otherwise, there shall be no member
```

Comment:
s/shall be/is/ since you're touching this line anyway?

Reply:
Makes sense to me. But then, should we fix `common_type` (probably not, since users can specialize it), `basic_common_reference` (ditto), `invoke_result` (probably should).

Reply:
Yes, we should fix those, but notably none of them are on this line. ;)